lms/djangoapps/support/static/support/jsx/entitlements/components/Main/MainContainer.jsx

```diff
@@ -0,0 +1,22 @@
+import { connect } from 'react-redux';
+
+import Main from './Main';
+
+const mapStateToProps = state => ({
+  entitlements: state.entitlements,
+  modal: state.modal
+})
+
+
+// Currently Main does not need any actionCreators
+// actionCreators will be added when Main needs them.
+function mapDispatchToProps(dispatch) {
```

Comment:
`ES6` no-op this by

```javascript
const mapDispatchToProps = () => {};
```

You could also just stick that no-op in the `connect`